config/rewrites.json

```diff
@@ -153,5 +153,148 @@
             "rewrite": false,
             "status": 302
         }
+    ],
+    "api.rackspace.com": [
+        {
+            "description": "Redirect root traffic to developer.rs.com/docs/",
+            "from": "^\\/$",
+            "to": "/docs/",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Servers redirect",
+            "from": "^\\/(api-ref(\\.html)?)?$",
+            "to": "/docs/cloud-servers/v2/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Servers extensions redirect",
+            "from": "^\\/api-ref-servers-ext(\\.html)?$",
+            "to": "/docs/cloud-servers/v2/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Auto Scale API v1.0 redirect",
+            "from": "^\\/api-ref-auto-scale(\\.html)?$",
+            "to": "/docs/autoscale/v1/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Backup redirect",
+            "from": "^\\/api-ref-backup(\\.html)?$",
+            "to": "/docs/cloud-backup/v1/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Big Data API v2 redirect",
+            "from": "^\\/api-ref-big-data-v2(\\.html)?",
+            "to": "docs/cloud-big-data/v2/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Block Storage redirect",
+            "from": "^\\/api-ref-blockstorage(\\.html)?",
+            "to": "/docs/cloud-block-storage/v1/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Databases redirect",
+            "from": "^\\/api-ref-databases(\\.html)?",
+            "to": "/docs/cloud-databases/v1/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud DNS redirect",
+            "from": "^\\/api-ref-dns(\\.html)?",
+            "to": "/docs/cloud-dns/v1/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Files redirect",
+            "from": "^\\/api-ref-files.html$",
```

Comment:
Does this need the optional-`html` bit applied, or is requiring the suffix intentional?

```
"^\\/api-ref-files(\\.html)?$"
```